src/api/app/models/package_issue.rb

```diff
@@ -2,6 +2,10 @@ class PackageIssue < ApplicationRecord
   belongs_to :package
   belongs_to :issue
 
+  scope :open_issues_with_owner, -> (owner_id) { joins(:issue).where(issues: { state: 'OPEN', owner_id: owner_id})}
+  scope :with_package_kinds, -> { joins('LEFT JOIN package_kinds ON package_kinds.package_id = package_issues.package_id') }
+  scope :with_kind, -> (kind) { where('package_kinds.kind = ?', kind) }
```

Comment:
do we want this in a scope? it will always be used with the join in the other scope :thinking: